### PR TITLE
Change formatting of arrays in mkxp.json comments

### PR DESCRIPTION
**Pull request by white-axe**
This changes the formatting of the multiline arrays in the comments in mkxp.json to put the opening bracket on its own line to improve readability.

### CODE REVIEW

File: mkxp.json [hunk 1]
```diff
@@ -150,7 +150,7 @@
     // A list of fonts to render without alpha blending.
     // (default: none)
     //
-    // "solidFonts": ["Arial", "Times New Roman"]
+    // "solidFonts": ["Arial", "Times New Roman"],
```
Comment (Splendide-Imaginarius) [inline mkxp.json]: Is there any reason you didn't split this one up into multiple lines?

Comment (white-axe) [reply mkxp.json]: It was already on one line. The splitting is mostly for the case where someone doesn't see that the first line doesn't have a closing bracket in the case of multiline arrays.

Comment (white-axe) [reply mkxp.json]: Whatever, I'll split this line too for consistency

**Comment by Splendide-Imaginarius** [comment]
This looks great by eye, thank you! Give me a little while to test this and make sure nothing got broken, but I expect to merge this.

**Comment by Splendide-Imaginarius** [comment]
Thanks!

**Pull request merged**